Add grid organization `list` and `show` commands

shannynalayna opened the pull request:
This change adds commands to list organizations and to show a specific organization based on the `org_id`. This also adds the man page entries for each of these commands.

dplumb94 reviewed (changes requested):
current output for --csv looks like this
```
ORG_ID,NAME,LOCATIONS,ALTERNATE_IDScrgl,Cargill,,gs1_company_prefix:0, asdf:asdfMacBook-Pro:grid dplumb$ 
```

I think to fix this it just needs a newline at the end of that output.

Also please add the list and show commands to the grid-organization.1.md man page in the See Also section

shannynalayna commented:
> current output for --csv looks like this
> 
> ```
> ORG_ID,NAME,LOCATIONS,ALTERNATE_IDScrgl,Cargill,,gs1_company_prefix:0, asdf:asdfMacBook-Pro:grid dplumb$ 
> ```
> 
> I think to fix this it just needs a newline at the end of that output.
> 
> Also please add the list and show commands to the grid-organization.1.md man page in the See Also section

@dplumb94 I fixed the list --csv option to look more like the way agents are listed in the csv format (with newlines after each row, and each individual value wrapped in quotes. I also added a commit to add links to the list/show man pages to the rest of the grid-organization-* man pages.
